src/client/components/forms/parts/achievement.tsx

```diff
@@ -57,8 +57,37 @@ const maxAchievementProgress = {
 	30: 100
 };
 
-function Achievement(props) {
-	const {achievement, counter, unlocked} = props;
+interface Props {
```

Comment:
Let's use a more precise name here to avoid confusion:

```suggestion
interface AchievementProps {
```